sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Metrics/PeriodFeedback.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.MetricsAdvisor.Models
+{
+    /// <summary> The PeriodFeedback. </summary>
+    public partial class PeriodFeedback : MetricFeedback
+    {
+        /// <summary>
+        /// The value of the feedback.
+        /// </summary>
+        public PeriodFeedbackValue Value { get; set; }
```

Comment:
I think we should hide `PeriodFeebackValue` and expose `PeriodType` and `PeriodValue` directly here.